### ISSUE TITLE
Chromecast Support?

### ISSUE THREAD

**Issue by LivInTheLookingGlass** [imshyam/mintube#16]
I don't know if this is possible in an open source app, but it would be very useful to have.

**Comment by danry25** [comment]
It is possible to add Chromecast support to a fair degree, akin to [stream2chromecast](https://github.com/Pat-Carter/stream2chromecast) or the [Butter Project](http://butterproject.org/), where a link to an HTTP(S) server is thrown at the chromecast to play. 

What isn't easily possible is multi-device shared youtube queues, a custom or styled Receiver Application or seamless playing of videos one after the other (instead there will be a slight delay at best). This is due to Google likely rejecting a custom [Receiver Application](https://developers.google.com/cast/docs/receiver_apps) (see flow chart), especially due to the way this project directly competes with the standard Youtube app.

TL;DR: Its totally possible, but it is unlikely to be akin to the Youtube App's chromecast integration.
